lib/main_common.pm

```diff
@@ -1162,10 +1162,9 @@ sub load_consoletests {
     if (!is_krypton_argon) {
         loadtest "console/curl_https";
     }
-    # puppet does not exist anymore in openSUSE Tumbleweed/Leap
-    if (is_sle && (check_var_array('SCC_ADDONS', 'asmm') && !sle_version_at_least('15'))
-        || (check_var_array('SCC_ADDONS', 'phub') && sle_version_at_least('15')))
-    {
+    # puppet does not exist anymore in openSUSE
```

Comment:
I was specific about openSUSE Tumbleweed/Leap because the package exists on build.opensuse.org. We could also state something like "official" but that might also not be clear. WDYT?